src/main/java/bookopedia/ui/PersonCard.java

```diff
@@ -54,26 +51,8 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
-        AtomicInteger parcelIndex = new AtomicInteger();
-        person.getParcels().stream()
-                .sorted(Comparator.comparing(parcel -> parcel.parcelName))
-                .forEach(parcel -> {
-                    Label label = new Label(parcelIndex.incrementAndGet() + ". " + parcel.parcelName);
-                    if (parcel.isFragile() && parcel.isBulky()) {
-                        label.getStyleClass().add("isFragileAndBulky");
-                    } else if (parcel.isFragile()) {
-                        label.getStyleClass().add("isFragile");
-                    } else if (parcel.isBulky()) {
-                        label.getStyleClass().add("isBulky");
-                    }
-                    parcels.getChildren().add(label);
-                });
```

Comment:
Maybe as a possible alternative, we can leave the parcels in the person list card at the right side too?

Reply:
Ok sure